src/Bounds.h

```diff
@@ -33,7 +33,8 @@ const FuncValueBounds &empty_func_value_bounds();
 Interval bounds_of_expr_in_scope(const Expr &expr,
                                  const Scope<Interval> &scope,
                                  const FuncValueBounds &func_bounds = empty_func_value_bounds(),
-                                 bool const_bound = false);
+                                 bool const_bound = false,
+                                 int indent = 0);
```

Comment:
Is this indent parameter only used internally? I don't think we should expose it.

Reply:
Yep. I just felt like the alternative (making an internal-only wrapper) was even uglier. Let me look again.

Reply:
PTAL